longestPalindrome.js

```diff
@@ -0,0 +1,27 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/longest-palindrome/
+
+const longestPalindrome = function (s) {
+  const hashMap = {};
```

Comment:
Lets rename to something more meaningful, I can see it's a **hash map**, I can't easily tell **what it maps into what** - either pick an appropriate name that reflects that or at least leave a comment like:

```js
const hashMap = {}; // maps symbol -> count where count is how often symbol is present in string s
```